aiohttp_session/nacl_storage.py

```diff
@@ -46,9 +57,14 @@ async def load_session(self, request):
                             "create a new fresh session")
                 return self.empty_session()
 
-    async def save_session(self, request, response, session):
+    async def save_session(
+        self,
+        request: web.Request,
+        response: web.StreamResponse,
+        session: Session
+    ) -> None:
         if session.empty:
-            return self.save_cookie(response, session._mapping,
+            return self.save_cookie(response, '',
```

Comment:
The reasoning is explained in the last bullet point of the MR description, sort version:

the `save_cookie` interface expects a string as the second argument see [here](https://github.com/aio-libs/aiohttp-session/blob/1127e80c039a876398ec18fc2c5d448229cf8214/aiohttp_session/cookie_storage.py#L51) for example.

The old code  worked as a side-effect, because `save_cookie` checks if `cookie_data` (the second argument) is 'Truthy' before using it (see [here](https://github.com/aio-libs/aiohttp-session/blob/1127e80c039a876398ec18fc2c5d448229cf8214/aiohttp_session/__init__.py#L243)) and if `session.empty` is `True` then `session._mapping` is an empty dictionary, which is not `Truthy` thus achieving the same result as an empty string, but violating the `save_cookie` type interface.